src/SquidConfig.h

```diff
@@ -542,6 +550,12 @@ class SquidConfig
         int connect_gap;
         int connect_timeout;
     } happyEyeballs;
+
+    using Id = InstanceId<SquidConfig>;
+    /// A Squid process (re)configuration identifier.
+    /// Changes with each lifecycleStart() call even if
+    /// reconfiguration changed nothing (that Squid could detect).
```

Comment:
Let's remove references to future features this code will eventually support and drop "re" prefix (because this data member identifies a given instance of Squid configuration).

```suggestion
    /// configuration ID that changes with each lifecycleStart() call
```